### PR TITLE
bugfix/0763178

### PR DESCRIPTION
**Pull request by cs-priyank-mankad**
Added Simulation in Navigation
Added Connector SOC in this zip

### CODE REVIEW

File: modules/comments/mmd.json [hunk 1]
```diff
@@ -9,14 +9,57 @@
     "userOwnable": false,
     "trackable": true,
     "taggable": true,
-    "peerReplicable": false,
+    "peerReplicable": true,
```
Comment (cs-pooja-singh) [inline modules/comments/mmd.json:12]: is this change intentional?

Comment (cs-priyank-mankad) [reply modules/comments/mmd.json:12]: I am doing this changes from MSSP system hence it is coming like this
